### DISCUSSION TITLE
How to install Box64 + Wine64 on Termux?

### DISCUSSION THREAD

**Discussion by GuckTubeYT** [ptitSeb/box64#458] [Q&A] [answered]
How to install Box64 + Wine64 on Termux?

**Comment by ptitSeb** [comment]
Converting this to a discussion.

**Comment by GuckTubeYT** [comment]
I just install Box64 + Wine64, but when i run `box64 wine64 explorer` i got the error like this

```
002c:err:virtual:virtual_setup_exception nested exception on signal stack in thread 002c addr 0x100047e05 stack 0x67fef580
0024:err:virtual:virtual_setup_exception nested exception on signal stack in thread 0024 addr 0x100047e05 stack 0x67fef580
```

**Comment by GuckTubeYT** [comment]
Uname -a = ```Linux localhost 5.10.101-android12-9-00001-gf4c0e37dbcde-ab8596533 #1 SMP PREEMPT Mon May 16 05:21:23 UTC 2022 aarch64 GNU/Linux```

**Comment by GuckTubeYT** [comment]
How do i fix that?

**Comment by GuckTubeYT** [comment, marked as answer]
Ok, i found how to fix that
You just type 
```
cmake .. -DARM_DYNAREC=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo
make -j 8
make install
```
and it Will fixed
